contribs/gnodev/main.go

```diff
@@ -128,7 +128,7 @@ func execDev(cfg *devCfg, args []string, io commands.IO) error {
 
 	// Setup Dev Node
 	// XXX: find a good way to export or display node logs
-	devNode, err := setupDevNode(ctx, rt, pkgpaths, gnoroot)
+	devNode, err := setupDevNode(ctx, rt, pkgpaths)
```

Comment:
The function signature of `setupDevNode` needs to be changed as well

Reply:
Updated, i've also added `make install.gnodev` to the CI to have a minimal build integration